### ISSUE TITLE
Add FILTEROFFSET documentation for NIRCam

### ISSUE THREAD

**Issue by stscijgbot-jp** [spacetelescope/jwst#6530]
_Issue [JP-2368](https://jira.stsci.edu/browse/JP-2368) was created on JIRA by [Rosa Diaz](https://jira.stsci.edu/secure/ViewProfile.jspa?name=rdiaz):_

Documentation for the reference file FILTEROFFSET used in the WCS step is missing for NIRCam. This should be added to read-the-docs.

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-2368?focusedCommentId=564752#comment-564752):_

Fixed by #6534

**Comment by stscijgbot-jp** [comment]
_Comment by [Rosa Diaz](https://jira.stsci.edu/secure/ViewProfile.jspa?name=rdiaz) on [JIRA](https://jira.stsci.edu/browse/JP-2368?focusedCommentId=564801#comment-564801):_

CRDS has CHANNEL, MODULE, USEAFTER as selection criteria for NIRCam's FILTEROFFSET. The new description in Read the Docs  (latest) lists  a different set for the selection criteria. Am I missing something?

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-2368?focusedCommentId=564808#comment-564808):_

You're absolutely right [Rosa Diaz](https://jira.stsci.edu/secure/ViewProfile.jspa?name=rdiaz). When I reviewed the PR I only looked for "mechanical" issues with the text changes, but didn't verify the accuracy of the information itself. NIRCam filteroffset files use INSTRUMENT, CHANNEL, MODULE, USEAFTER to select the ref files from CRDS. The narrowing down to FILTER+PUPIL values is done by the step itself when searching through the multiple rows of info contained within the ref file. So this definitely needs to be changed in the docs.
